boot.php

```diff
@@ -767,7 +767,7 @@ function __construct() {
 
 	}
 
-	function get_basepath() {
+	public static function get_basepath() {
```

Comment:
Can we assume that you checked this function is now only uses statically at any place?